cleanlab/classification.py

```diff
@@ -251,10 +211,16 @@ def fit(
         thresholds=None,
         noise_matrix=None,
         inverse_noise_matrix=None,
+        find_label_issues_kwargs = {"filter_by": "prune_by_noise_rate",
+                                    "min_examples_per_class": 10},
```

Comment:
Could you elaborate on this?  

The user can just solely specify `find_label_issues_kwargs={'min_examples_per_class': 5}` and the other args will be left at their defaults. These are just optional kwargs, so any kwarg not in this dict will be left at its default value in `find_label_issues()`.  I've added clarifying sentence to docstring:

"Arguments not specified here will remain at their default values."

Reply:
Also FYI: in the default setting of `find_label_issues_kwargs` I've set:
`"filter_by": "prune_by_noise_rate"` which is unnecessary. I did this to encourage users to play with it as it may be important hyperparameter, and it provides more of an example of how to use `find_label_issues_kwargs`. We can also remove this is you strongly prefer?

Reply:
This patch currently has the weird/unintuitive behavior that if you don't pass `find_label_issues_kwargs`, it will use the default argument (214--215 above), which includes `min_examples_per_class=10`, but if you pass e.g. `find_label_issues_kwargs={'filter_by': 'both'}`, when it calls `find_label_issues`, it won't pass `min_examples_per_class` and hence use _its_ default value for that argument, which is different (it's set to `1` in this PR: https://github.com/cleanlab/cleanlab/pull/149/commits/e1d9ae4fcf95e802f4a8e112845929ee785705eb#diff-9b65a96b3894947ee2eee0fd92220c97695cc9cb72a27fb256cd6fe4d31e975bR245).

Reply:
I see your point, what's your preferred way to resolve this?  They all seem to have other tradeoffs.

Reply:
Couple variations on exactly how to write it. One example:

```python
DEFAULT_FIND_LABEL_ISSUES_KWARGS = {"filter_by": "prune_by_noise_rate", "min_examples_per_class": 10}}
# document that the above is effectively the default, but you can override it component-wise
def fit(...
    find_label_issues_kwargs = {}
):
    find_label_issues_kwargs = {**DEFAULT_FIND_LABEL_ISSUES_KWARGS, **find_label_issues_kwargs}
```

Reply:
Ya that's what I was thinking, don't think users will be confused by (or even notice) this.